### PR TITLE
updates for root master api changes

### PR DESCRIPTION
**Pull request by roblatham00**
Not sure what version of ROOT i'm supposed to target.  root@master has made a few api changes, which I've attempted to track with this pull request.

Review requested of the RNTupleView change -- wasn't sure which boolean value to use there.

### CODE REVIEW

File: SerialRNTupleSource.h [hunk 1]
```diff
@@ -46,7 +47,7 @@ namespace cce::tf {
   private:
     void fillViews(ROOT::Experimental::RNTupleReader& iReader, std::vector<std::string> const& iFieldIDs);
     SerialTaskQueue* queue_;
-    std::vector<ROOT::Experimental::RNTupleView<void>> views_;
+    std::vector<ROOT::Experimental::RNTupleView<void, false>> views_;
```
Comment (pcanal) [inline SerialRNTupleSource.h:50]: The new declaration is: template `template <typename T, bool UserProvidedAddress> RNTupleView`.   Is the mimicking framework providing to RNtuple or getting from RNTuple the object addresses?

**Comment by Dr15Jones** [comment]
I actually have the needed compilation changes sitting on the disk at work. I'll be sure to push my full changes.

**Comment by Dr15Jones** [comment]
> Review requested of the RNTupleView change -- wasn't sure which boolean value to use there.

You did pick the correct bool for this code. In the PR I just made, I also found I needed to change two `#include` statements as well.

If it is OK with you, I was going to merge my similar PR instead of yours.

**Comment by Dr15Jones** [comment]
Ah, I should have looked at this PR fully. It is identical to what I have so I'll merge this one!

**Comment by roblatham00** [comment]
Of course.. was entirely expecting the proper approach to be something
different

**Pull request merged**